### PR TITLE
add untrusted rp

### PR DESCRIPTION
**Pull request by vladikoff**
@dannycoates @jrgm feedback?

Don't merge yet...

### CODE REVIEW

File: aws/cloudformation/moz-single.json [hunk 1]
```diff
@@ -365,6 +365,14 @@
             }
           },
           {
+            "Name" : { "Fn::Join" : [ "", ["321done-untrusted-", {"Ref" : "Subdomain"}, ".", {"Ref" : "HostedZone"}, "."]]},
```
Comment (jrgm) [inline aws/cloudformation/moz-single.json]: In the hostname, and later in the url path /123done-untrusted, I would drop the `-untrusted` part. Just seems too fugly.

**Comment by jrgm** [comment]
You can ignore my nit, if you don't agree. Otherwise, LGTM.

**Comment by vladikoff** [comment]
> You can ignore my nit, if you don't agree. Otherwise, LGTM.

As long as we don't get confused what 123 vs 321 is. We can update the index.html later as well.

Updated. running `make` again.

**Comment by vladikoff** [comment]
merging this now . @dannycoates if you find any issues let me know and I will update this in a separate commit.

**Pull request merged**